org.eclipse.scout.rt.ui.html/src/main/java/org/eclipse/scout/rt/ui/html/json/UploadRequestHandler.java

```diff
@@ -359,6 +369,18 @@ protected void verifyFileName(Set<String> validFileExtensions, String filename,
     }
   }
 
+  /**
+   * Checks if the uploaded file exceeds the maximum allowed upload size for given {@code uploadable}.
+   *
+   * @throws RejectedResourceException
+   *           when size of part exceeds the allowed upload size
+   */
+  protected void verifyMaximumUploadSize(IUploadable uploadable, Part part) {
+    if (part.getSize() > uploadable.getMaximumUploadSize()) {
+      throw new RejectedResourceException("The field {} exceeds its maximum permitted size of {} bytes.", part.getName(), uploadable.getMaximumUploadSize());
```

Comment:
Can we rely on part.getSize? The previous implementation did check the provided size but also created a LimitedInputStream which limited the actual number of bytes read from the stream to the maximum size.

Reply:
part.getSize() seems to be reliable, at least within my local tests.

Additionally we could also limit the IOUtility call: 

```
content = IOUtility.readBytes(in, (int) uploadable.getMaximumUploadSize());
```
This would lead to a similar limitation as the former LimitedInputStream.
Drawback: Implementation in IOUtilty seems a little bit less performant when a limit size is used.